src/read.c

```diff
@@ -2907,8 +2854,6 @@ static StructInitInfo module = {
     .name = "read",
     .initKernel = InitKernel,
 
-    .moduleStateSize = sizeof(struct ReaderState),
-    .moduleStateOffsetPtr = &ReaderStateOffset,
```

Comment:
There is no per-thread `ReaderState` as part of the `GAPState` anymore, so the above can go.